arci-speak-cmd/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "arci-speak-cmd"
-version = "0.1.0"
+version = "0.0.1"
```

Comment:
arci-speak-cmd v0.1.0 is already released.
(Almost by mistake)
Let's keep 0.1.0.

Reply:
changed version of arci-speak-cmd to 0.1.0